test/torturethread.c

```diff
@@ -63,7 +63,7 @@ ThreadFunc(void *data)
 
     SDL_Log("Thread '%d' waiting for signal\n", tid);
     while (SDL_GetAtomicInt(&time_for_threads_to_die[tid]) != 1) {
-        ; /* do nothing */
```

Comment:
These semi-colons in an otherwise empty body fix the `-Wempty-body` gcc warning. Can you remove those changes from this PR?

Reply:
Removed them.

Reply:
Just tested all variations:
`if` / `while`
`{;}` / `{}` / `;`
I only get the warning with the `if` statement and a semicolon: `if();`
All other combinations seem to be fine with it.